hack/quick_install.sh

```diff
@@ -68,10 +68,8 @@ helm template --namespace=istio-system \
 kubectl apply -f istio-local-gateway.yaml
 
 # Install Knative
-kubectl apply --selector knative.dev/crd-install=true \
---filename https://github.com/knative/serving/releases/download/${KNATIVE_VERSION}/serving.yaml \
-
-kubectl apply --filename https://github.com/knative/serving/releases/download/${KNATIVE_VERSION}/serving.yaml \
+kubectl apply --filename https://github.com/knative/serving/releases/download/${KNATIVE_VERSION}/serving-crds.yaml
+kubectl apply --filename https://github.com/knative/serving/releases/download/${KNATIVE_VERSION}/serving-core.yaml
 
```

Comment:
Does this separation of yamls mean also a new mnimum version of KNative serving?

Reply:
No, the previous installation has a race condition, so here is to fix using the correct way according to the [knative doc here](https://knative.dev/docs/install/any-kubernetes-cluster), also this quick install is only for demonstration purpose.